src/vcpkg/commands.build.cpp

```diff
@@ -1561,17 +1582,25 @@ namespace vcpkg
                        "**Failure logs**\n\n```\n{}\n```\n",
                        paths.get_filesystem().read_contents(build_result.stdoutlog.value_or_exit(VCPKG_LINE_INFO),
                                                             VCPKG_LINE_INFO));
-        result.append(Strings::join("\n", build_result.error_logs, CreateLogDetails{fs}));
 
+        std::string postfix;
         const auto maybe_manifest = paths.get_manifest();
         if (auto manifest = maybe_manifest.get())
         {
-            fmt::format_to(
-                std::back_inserter(result),
-                "**Additional context**\n\n<details><summary>vcpkg.json</summary>\n\n```\n{}\n```\n</details>\n",
-                Json::stringify(manifest->manifest));
+            if (args.get_command() != "x-set-installed" || manifest->manifest.contains("builtin-baseline"))
```

Comment:
I am not OK with reaching into args and guessing whether something is relevant like this.

Reply:
Ok I will add something like `executed_from_install_command` to `install_execute_plan`